images: implemented image resizing

FaiyazMujawar opened the pull request:
Fixes #6 

In this PR,
- implemented functionality to resize images before saving on the disk.
- The images are resized to default size of 1366x720p

drigio commented on routes/file.js:
```diff
@@ -1,15 +1,46 @@
 
+/** Size to which the image is to be resized */
+const IMAGE_SIZE = {
+  height: 1080,
```
Make height `1366`

drigio commented on routes/file.js:
```diff
@@ -1,15 +1,46 @@
+      // Resizing image
+      const buffer = await sharp(path)
+        .resize(IMAGE_SIZE.height, IMAGE_SIZE.width)
+        .jpeg({ quality: 90 })
```
reduce quality even more. We want less size of file.

FaiyazMujawar replied:
Specify a value for quality

drigio replied:
Make it 60-70 maybe

FaiyazMujawar committed:
images: implemented image resizing

In this PR,
- implemented functionality to resize images before saving on the disk.
- The images are resized to default size of 1366x720p

drigio commented:
LGTM! Merging in.
